stdlib/public/runtime/Private.h

```diff
@@ -183,6 +183,8 @@ namespace swift {
                              void *data);
   uint8_t *_swift_getSectionDataPE(void *handle, const char *sectionName,
                                    unsigned long *sectionSize);
+#endif
+#if defined(__CYGWIN__)
   void _swift_once_f(uintptr_t *predicate, void *context,
```

Comment:
```sizeof(swift_once_t) <= sizeof(void*)``` was passed. But I couldn't conform that  ```std::call_once()``` follows compatible init process (0 to 1 to 2). Could you let me know the counter part source which determines the compatibility.